dir.targets

```diff
@@ -95,7 +95,7 @@
 
     <!-- Download latest nuget.exe -->
     <DownloadFile FileName="$(NuGetToolPath)"
-                  Address="https://dist.nuget.org/win-x86-commandline/v3.2.0-rc/nuget.exe"
+                  Address="https://dist.nuget.org/win-x86-commandline/v3.2.0/nuget.exe"
```

Comment:
I think either way would be OK.  Using a specific version means it's more likely we'll get the same behavior if we try to build at a previous point in the commit history.  However, using the latest version means we don't have to update it, and is the pattern that was used for NuGet v2.